Makefile

```diff
@@ -31,6 +31,15 @@ build: ## Builds binary package
 	@go version | grep -q -E "go${GOLANG_VERSION_REGEX} " || (echo "Required Go version is ${GOLANG_VERSION}\nInstalled: `go version`" && exit 1)
 	go build -v -ldflags "-X main.Version=${VERSION} -X main.GitRev=${GITREV} -X main.BuildDate=${BUILD_DATE}" .
 
+.PHONY: debug
+debug: ## Builds binary package
```

Comment:
I think it would be better to reuse the same build target and modify it's behaviour using variables.

Here is an example for that:

https://github.com/sagikazarmark/modern-go-application/blob/master/Makefile

Fixing this can be part of a subsequent PR though